provisioning/provisioning_info.go

```diff
@@ -14,7 +14,7 @@ type NetworkStackType int
 const (
 	NetworkStackV4   NetworkStackType = 1 << iota
 	NetworkStackV6   NetworkStackType = 1 << iota
-	NetworkStackDual NetworkStackType = (NetworkStackV4 & NetworkStackV6)
+	NetworkStackDual NetworkStackType = (NetworkStackV4 | NetworkStackV6)
```

Comment:
groan, how did I do this :facepalm:

Reply:
I read this earlier and also did not notice.